doc/ARCHITECTURE.md

```diff
@@ -1,29 +1,55 @@
+### Version control
+The code base, hosted on GitHub and maintained by the Deven team, is publicly accessible, inviting interested users to contribute to the ongoing development and maintenance efforts of the project. We encourage collaboration and welcome feedback from the community to enhance the project's capabilities and ensure its continued evolution.
 
-## Technical Decisions
+### Documentation and Testing
+The project adheres to best practices for documentation, and testing. The codebase is thoroughly documented with live preview capability and explicit guidelines on usage and contribution procedures. Additionally, comprehensive Component tests are integrated to verify that modifications uphold the project's integrity and functionality, preventing regressions or defects from emerging.
 
+### CI-CD
+The changes made in project's code base adheres to a continuous integration and delivery (CI/CD) pipeline. This guarantees that any alterations undergo rigorous testing and validation before being released to the production environment.
+
+## Technical decision
 In this section you should list all technical things you use like frameworks, tools, languages and provide explanations of why you have choosen them. Think about e.g. what are the main reasons you choose a framework, what are the advatages you need for your project? Try to be as clear as possible so also new joiners understand why your setup is the best for your project.
 
-## APIs
+### Web components
+Our design system includes a wide range of reusable UI components, each carefully designed and implemented using the [Lit](https://lit.dev/docs/). These web components cover various aspects of our application's user interface, including buttons, input fields, tab bars, divider, and more. The reason for using Lit was that it being a straightforward library designed for constructing rapid, lightweight web components. Furthermore, it provides the developers a component base class that eliminates boilerplate code, offering reactive state, scoped styles, and a declarative template system. This system is characterized by its compact size, swift performance, and expressive nature.
+
+### Styles
+The design system provides a comprehensive set of styles, including typography, colors, spacing, and layout guidelines. These styles are meticulously crafted to maintain visual coherence and enhance the overall user experience. SEE WHAT CAN BE ADDED FROM FIGMA OR PROJECT INTRO PAGE.
+
+### Events
+The various events, such as focus, blur, click, etc., have the 'blr' prefix appended, allowing them to be accessed as 'blrFocus,' 'blrBlur,' 'blrClick,' and so on. Detail can be found in [Storybook documentation](https://github.com/deven-org/B01LER-Kitchen).
```

Comment:
Are there more details in the Storybook documentation about this? Also the link is leading to the Repo and not to Storybook.